manifests/updater.pp

```diff
@@ -5,13 +5,13 @@
 ) inherits bind::defaults {
 
     if $nsupdate_package {
-        package { 'nsupdate':
+        package { 'bind-tools':
             ensure => present,
             name   => $nsupdate_package,
         }
     }
 
-    class { 'bind::keydir':
-        keydir => $keydir,
-    }
+    # class { 'bind::keydir':
```

Comment:
`class { 'bind::keydir': ... }` needs to be in `bind::updater` so that `bind::key` definitions will still work on non-servers.